robot_modeling/DQ_SerialManipulator.m

```diff
@@ -1,30 +1,27 @@
-% Concrete class that defines serial manipulators.
+% Abstract class that defines serial manipulators.
 %
-% Usage: robot = DQ_SerialManipulator(A,convention)
-% - 'A' is a 4 x n matrix containing the Denavit-Hartenberg parameters
-%   (n is the number of links)
-%    A = [theta1 ... thetan;
-%            d1  ...   dn;
-%            a1  ...   an;
-%         alpha1 ... alphan]
-% - 'convention' is the convention used for the D-H parameters. Accepted
-%    values are 'standard' and 'modified'
-%
-% The first row of 'A' contains the joint offsets. More specifically,
-% theta_i is the offset for the i-th joint.
+% Usage: robot = DQ_SerialManipulator(dim_configuration_space)
```

Comment:
@juanjqo, please remove those lines because the constructor does not have any parameters in the current implementation.

Reply:
@bvadorno sure!
[Implemented](https://github.com/dqrobotics/matlab/pull/75/commits/91f00ca4896b044b6b9cf99ae7382f056a40aa7a).